common/processors/init_physic_processor.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from common.engine.processor import Processor
+from common.engine.resource_manager import ResourceManager
+from common.resources.physic_resource import PhysicResource
+
+logger = logging.getLogger(__name__)
+
+
+class InitPhysicProcessor(Processor):
+    def __init__(self):
+        super().__init__()
+
+    def process(self, r: ResourceManager) -> None:
+        world: PhysicResource = r.get_resource(PhysicResource)
```

Comment:
`def get_resource(self, resource: type[R]) -> R`

```suggestion
        world = r.get_resource(PhysicResource)
```